translations/en-us.yml

```diff
@@ -1845,6 +1845,11 @@ osf-components:
         delete_file: 'Delete file'
         delete_folder: 'Delete folder'
         delete_fail: 'Failed to delete "{fileName}"'
+        errors:
+            load_file_list: 'Failed to load files'
+            load_file_provider: 'Failed to load the file provider'
+            load_file_provider_title: 'Provider error'
+            load_file_providers: 'Failed to load list of addon providers'
```

Comment:
Are there fixes or other info that we can suggest for users? Having the toast show up is a huge improvement from what is there now, but when I run into this issue, it's not very reliable about showing me folders/files until I reload the page.

Reply:
That's true. A lot of ways for the files list request to fail. I'd be ok with leaving the messages as they are if there isn't an obvious thing for users to try.